0002-gitops-management-clusters/0002-gitops-management-clusters.md

```diff
@@ -0,0 +1,27 @@
+# RFC 0002 - Enable customers to use gitops in management clusters
+
+We currently give customers access to the management clusters but do not support them in utilizing this access effectively in terms of git ops related management (e.g. for apps).
+
+Context:
+Some customers might be interested in utilizing mainstream git ops tooling in order to manage their apps across clusters and installations.
+
+## Open Questions
+
+### 1. General direction
+1.1 We would like to allow our customers to choose their management tooling freely. Do we want to support some tools with e.g. a managed app first though?
```

Comment:
We want to let customers choose their tooling freely but I think we need some validation so only a subset of apps can be installed in management clusters. 

Starting with a managed app first makes sense to me. Especially if its a tool we choose to use ourselves.

Reply:
I agree that managed apps makes sense. It's also a question of how much freedom we give customers / how much we allow them to "fuck up".

Reply:
It could be that we limit this for the beginning to there's this selection of apps (at the get go 1) that you can install on your MC using our interface. very controlled.

Later, it could be extended to having nicely isolated namespaces in the MC where we can give a certain amount of more freedom, but even there because of the limits of isolation we need to be very careful and we might never get there completely.

Reply:
because it's not only about "fuckign up" it might also be that there's cases of actual multi-tenancy within an MC, where there's an untrusted org